pkg/systemd/services/services.jsx

```diff
@@ -151,6 +151,50 @@ class ServicesPage extends React.Component {
             tabErrors: {},
             isFullyLoaded: false,
         };
+
+        // Possible LoadState values: stub, loaded, not-found, bad-setting, error, merged, masked
+        // See: typedef enum UnitLoadStateState https://github.com/systemd/systemd/blob/main/src/basic/unit-def.h
+        this.loadState = {
+            stub: _("Stub"),
+            loaded: "",
```

Comment:
Why we don't represent this one?

Reply:
Firstly, this did not change. 
If you ask generally - we don't specify the difference in the UI between loaded and unloaded units. We just show everything.

Reply:
Right, I also noticed that we have checked for `loaded` in render method.